### DISCUSSION TITLE
Drawio desktop for corporate use

### DISCUSSION THREAD

**Discussion by emilioluiz** [jgraph/drawio#4666] [General]
Good afternoon. I work for a large company in Brazil, but they have blocked the use of Drawio for all employees. They are claiming that they need a response from jgraph confirming whether Drawio is truly free for corporate use, without requiring any license. I need help in responding to them so that we can get the software unblocked and use it here. If anyone can help, I would be very grateful

**Comment by davidjgraph** [comment]
draw.io can be used at zero cost, downloaded from this repo, under the Apache 2.0 license. You cannot use any kind of software "without requiring any license".

**Reply by emilioluiz** [reply]
Thanks @davidjgraph
